_config/config.yml

```diff
@@ -17,3 +17,5 @@ HTTP:
     must-revalidate: "true"
     no-transform: "true"
   vary: "Cookie, X-Forwarded-Protocol, User-Agent, Accept"
+Hierarchy:
+  node_threshold_total: 30
```

Comment:
This is the framework module, why should the framework set a default for it's own class to two different values in two places?

```
private static $node_threshold_total = 50;
```

This PR changes the default to 30. This above 50 is now redundant.